src/app/beneficiaries/create.jsx

```diff
@@ -1,192 +1,441 @@
+						<label htmlFor="dni" className="hidden md:block text-black">
+							DNI
+						</label>
+						<div className="relative flex items-center border-2 rounded-xl border-gray-200 bg-white">
+							<svg
+								xmlns="http://www.w3.org/2000/svg"
+								fill="none"
+								viewBox="0 0 24 24"
+								strokeWidth="1.5"
+								stroke="currentColor"
+								className="w-4 h-4 absolute left-0 m-1"
+							>
+								<path
+									strokeLinecap="round"
+									strokeLinejoin="round"
+									d="M15 9h3.75M15 12h3.75M15 15h3.75M4.5 19.5h15a2.25 2.25 0 0 0 2.25-2.25V6.75A2.25 2.25 0 0 0 19.5 4.5h-15a2.25 2.25 0 0 0-2.25 2.25v10.5A2.25 2.25 0 0 0 4.5 19.5Zm6-10.125a1.875 1.875 0 1 1-3.75 0 1.875 1.875 0 0 1 3.75 0Zm1.294 6.336a6.721 6.721 0 0 1-3.17.789 6.721 6.721 0 0 1-3.168-.789 3.376 3.376 0 0 1 6.338 0Z"
 								/>
-							</div>
+							</svg>
+							<input
+								className="flex items-center rounded-xl p-1 pl-6 w-full"
+								type="text"
+								placeholder="DNI"
+								id="dni"
+								name="dni"
+							/>
 						</div>
-
-						<div>
-							<label htmlFor="firstTimeAttended">
-								<strong>Primera atención</strong>
-							</label>
-							<div className="flex items-center border-2 rounded-md border-gray-200 bg-white">
-								<input
-									type="date"
-									name="firstTimeAttended"
-									className="p-1 w-full"
+					</fieldset>
+					<fieldset className="flex flex-col w-full md:w-5/12">
+						<label
+							htmlFor="dossier_number"
+							className="hidden md:block text-black"
+						>
+							Número de registro
+						</label>
+						<div className="relative flex items-center border-2 rounded-xl border-gray-200 bg-white">
+							<svg
+								xmlns="http://www.w3.org/2000/svg"
+								fill="none"
+								viewBox="0 0 24 24"
+								strokeWidth="1.5"
+								stroke="currentColor"
+								className="w-4 h-4 absolute left-0 m-1"
+							>
+								<path
+									strokeLinecap="round"
+									strokeLinejoin="round"
+									d="M15 9h3.75M15 12h3.75M15 15h3.75M4.5 19.5h15a2.25 2.25 0 0 0 2.25-2.25V6.75A2.25 2.25 0 0 0 19.5 4.5h-15a2.25 2.25 0 0 0-2.25 2.25v10.5A2.25 2.25 0 0 0 4.5 19.5Zm6-10.125a1.875 1.875 0 1 1-3.75 0 1.875 1.875 0 0 1 3.75 0Zm1.294 6.336a6.721 6.721 0 0 1-3.17.789 6.721 6.721 0 0 1-3.168-.789 3.376 3.376 0 0 1 6.338 0Z"
 								/>
-							</div>
+							</svg>
+							<input
+								className="flex items-center rounded-xl p-1 pl-6 w-full"
+								type="text"
+								placeholder="Numero de registro"
+								id="dossier_number"
+								name="dossier_number"
+							/>
 						</div>
+					</fieldset>
+					<fieldset className="flex flex-col w-full md:w-5/12">
+						<label htmlFor="birth-date" className="text-black">
+							Fecha de nacimiento
+						</label>
+						<div className="relative flex items-center border-2 rounded-xl border-gray-200 bg-white">
+							<input
+								className="flex items-center rounded-xl p-1 w-full"
+								type="date"
```

Comment:
To create an intervention, it must have an input type datetime and no input type date, since both are registered.